src/main/groovy/com/ullink/gradle/opencover/OpenCover.groovy

```diff
@@ -91,9 +77,11 @@ class OpenCover extends ConventionTask {
         def commandLineArgs = getCommonOpenCoverArgs()
 
         if (!getParallelForks() || getParallelTargetExecArgs().size() == 0) {
+            coverageReportPath = new File(reportsFolder, "coverage.xml")
             runSingleOpenCover(commandLineArgs)
         }
         else {
+            coverageReportPath = reportsFolder.path.toString() + File.separator + "*.xml"
```

Comment:
having a wildcard in the property may be confusing (users will have to check if the name contains a wildcard before trying to open/read the file), can't we just set the folder path instead ?

Reply:
We need to specify the path this way so that sonar will know how to load multiple coverage report files: https://docs.sonarqube.org/pages/viewpage.action?pageId=6389770#CodeCoverageResultsImport(C#,VB.NET)-OpenCover